Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: We don't need to use any release candidates in debian.  Keep it simple.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -116,7 +116,7 @@
 optional = true
 
 [target.'cfg(target_os = "linux")'.dependencies.dbus-secret-service]
-version = "4.0.0-rc.2"
+version = "4.0.1"
 optional = true
 
 [target.'cfg(target_os = "linux")'.dependencies.linux-keyutils]
@@ -133,7 +133,7 @@
 optional = true
 
 [target.'cfg(target_os = "openbsd")'.dependencies.dbus-secret-service]
-version = "4.0.0-rc.1"
+version = "4.0.1"
 optional = true
 
 [target.'cfg(target_os = "openbsd")'.dependencies.secret-service]
